src/core/core.controller.js

```diff
@@ -998,18 +1009,20 @@ class Chart {
 	}
 
 	/**
+	 * @param {IEvent} e
+	 * @param {boolean} [replay]
 	 * @private
 	 */
-	eventHandler(e) {
+	eventHandler(e, replay) {
```

Comment:
The boolean is called `useFinalPosition` in the handlers, so perhaps we should use that too here. Should also document what it does in the comment here and in the plugin docs that we ship

Reply:
I opted to do `const useFinalPosition = replay;` in `_handleEvent` and `afterEvent`